cmd/fund/fund.go

```diff
@@ -0,0 +1,132 @@
+package fund
+
+import (
+	"crypto/ecdsa"
+	"fmt"
+	"log"
+	"math/big"
+
+	_ "embed"
+
+	"github.com/chenzhijie/go-web3"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/spf13/cobra"
+)
+
+var (
+	//go:embed usage.md
+	usage                string
+	walletCount          int
+	fundingWalletAddress string
+	fundingWalletPK      string
+	chainID              int
+	chainRPC             string
+	walletFundingAmt     float64
+	walletFundingGas     uint64
+)
+
+func generateWalletAddresses(numWallets int) ([]*common.Address, error) {
+	var addresses []*common.Address
+
+	for i := 0; i < numWallets; i++ {
+		account, err := crypto.GenerateKey()
+		if err != nil {
+			log.Fatal(err)
+		}
+		addr := crypto.PubkeyToAddress(account.PublicKey)
+		addresses = append(addresses, &addr)
+	}
+	return addresses, nil
+}
+
+func fundWallets(web3Client *web3.Web3, wallets []*common.Address, senderAddress common.Address, senderPrivateKey *ecdsa.PrivateKey, amountWei *big.Int, walletFundingGas uint64) error {
+	nonce, err := web3Client.Eth.GetNonce(senderAddress, nil)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	for _, wallet := range wallets {
+		_, err := web3Client.Eth.SyncSendRawTransaction(
+			common.HexToAddress((*wallet).Hex()),
+			amountWei,
+			nonce,
+			walletFundingGas,
+			web3Client.Utils.ToGWei(1),
+			nil,
+		)
+		if err != nil {
+			log.Fatal(err)
+		}
+		fmt.Println("Funded", (*wallet).Hex(), "with", amountWei, "wei")
```

Comment:
nit: switch to `github.com/rs/zerolog/log` for logging to leverage the global `verbosity` flag.